iis3dwb_reg.c

```diff
@@ -1727,7 +1727,7 @@ int32_t iis3dwb_i2c_interface_get(stmdev_ctx_t *ctx,
 {
   iis3dwb_ctrl4_c_t ctrl4_c;
 
-  conmst int32_t ret = iis3dwb_read_reg(ctx, IIS3DWB_CTRL4_C, (uint8_t *)&ctrl4_c, 1);

```

Comment:
Still I don not get the reason for this as already commented elsewhere.
But in any case, can you squash this commit with your previous one?

Reply:
squashed